docs/source/userguide/command_line.rst

```diff
@@ -261,21 +290,40 @@ For an example of how to run this command, see below.
 
     geoips list test-datasets
 
+``list unit-tests`` is a list sub-command which retrieves a listing of unit tests from
+all, or a certain GeoIPS Package. For this command to find your unit tets, you must
+place the unit tests under ``<package_name>/tests/unit_tests/``. These test scripts can
+then be ran using ``pytest -v /path/to/<package_name/tests/unit_tests/<unit_test_dir>``.
+This command can only be ran if the specified plugin package[s] are installed in
+*editable* mode.
```

Comment:
Does it have to be in editable mode? Or does the repo just have to be cloned (ie, could clone the repo, then install in site-packages ?)

It doesn't really matter, just asking out of curiosity / for my own understanding.

Just respond, and resolve this conversation.

Reply:
It only works when it's in editable mode, since we are interrogating the /tests... directory. If it's installed in site-packages, this won't work since we don't include that directory. Hope this explains things!